### PR TITLE
"Getting started gce" deadlink

### PR DESCRIPTION
**Pull request by MasayaAoyama**
1.
modify for kubernetes/website link

2.
remove unnecessary .md link

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.11 Features: set Milestone to 1.11 and Base Branch to release-1.11
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> NOTE: After opening the PR, please *un-check and re-check* the ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) box so that maintainers can work on your patch and speed up the review process. This is a temporary workaround to address a known issue with GitHub.> 
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

### CODE REVIEW

File: content/en/blog/_posts/2016-07-00-Autoscaling-In-Kubernetes.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ To understand better where autoscaling would provide the most value, let’s sta
 
 The following instructions apply to GCE. For GKE please check the autoscaling section in cluster operations manual available [here](https://cloud.google.com/container-engine/docs/clusters/operations#create_a_cluster_with_autoscaling).  
 
-Before we begin, we need to have an active GCE project with Google Cloud Monitoring, Google Cloud Logging and Stackdriver enabled. For more information on project creation, please read our [Getting Started Guide](https://github.com/kubernetes/kubernetes/blob/master/docs/getting-started-guides/gce.md#prerequisites). We also need to download a recent version of Kubernetes project (version [v1.3.0](http://v1.3.0/) or later).  
+Before we begin, we need to have an active GCE project with Google Cloud Monitoring, Google Cloud Logging and Stackdriver enabled. For more information on project creation, please read our [Getting Started Guide](https://github.com/kubernetes/website/blob/master/content/en/docs/getting-started-guides/gce.md#prerequisites). We also need to download a recent version of Kubernetes project (version [v1.3.0](http://v1.3.0/) or later).
```
Comment (justinsb) [inline content/en/blog/_posts/2016-07-00-Autoscaling-In-Kubernetes.md]: Not sure of our policy here, but should we link to https://kubernetes.io/docs/getting-started-guides/gce/ (i.e. the generated form of the website) ?

**Comment by MasayaAoyama** [comment]
@justinsb 
Thank you for your review.
I think it same, so I modify the link from github.com to kubernetes.io
Best,

**Comment by MasayaAoyama** [comment]
I heard from mistyhacks, HUGO migration is checked by you.
Thank you for your help!

/assign @steveperry-53
/assign @Bradamant3

**Comment by Bradamant3** [comment]
@MasayaAoyama we don't accept PRs against old blog posts (take a look at the date ;) ). The change you made to the other file is useful, though. Could you please remove the blog post file and resubmit for the other change? Thanks!

**Comment by MasayaAoyama** [comment]
@Bradamant3 Hello, Thank you for your review.
I modified for your suggestion.

**Pull request merged**